tests/dimension_validation_schema.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Software Quality Dimension",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "type": "string",
+            "const": "https://w3id.org/everse/rsqd#"
+        },
+        "@id": {
+            "type": "string",
+            "format": "uri"
+        },
+        "@type": {
+            "type": "string",
+            "const": "SoftwareQualityDimension"
+        },
+        "name": {
+            "type": "string",
+            "description": "The name of the quality dimension."
+        },
+        "description": {
+            "type": "string",
+            "description": "A detailed description of the quality dimension."
+        },
+        "identifier": {
+            "type": "string",
+            "format": "uri",
+            "description": "A unique identifier for the quality dimension."
+        },
+        "keywords": {
+            "type": "array",
+            "items": { "type": "string" },
+            "description": "Keywords describing the quality dimension."
+        },
+        "author": {
+            "type": "string",
+            "format": "uri",
+            "description": "The author or creator of the quality dimension."
+        },
+        "created": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The date this quality dimension was created."
+        },
+        "version": {
+            "type": "string",
+            "description": "Version number of the quality dimension definition."
+        },
+        "contact": {
+            "type": "string",
+            "format": "uri",
+            "description": "Contact point for further inquiries."
+        },
+        "source": {
+            "type": "object",
+            "description": "Source of the quality dimension definition.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "Name of the source."
+
+                },
+                "url": {
+                    "type": "string",
+                    "description": "URL of the source."
+                }
+            },
+            "required": [
+                "name",
+                "url"
+            ]
+        }
```

Comment:
There is a property missing `isPartOf` to allow stating that a dimension is part of another dimension

Reply:
This was introduced in 0.0.2 after the GAM conversations